src/main/sensors/gyro.c

```diff
@@ -417,7 +417,9 @@ static bool gyroDetectSensor(gyroSensor_t *gyroSensor, const gyroDeviceConfig_t
  || defined(USE_ACC_MPU6050) || defined(USE_GYRO_SPI_MPU9250) || defined(USE_GYRO_SPI_ICM20601) || defined(USE_GYRO_SPI_ICM20649) || defined(USE_GYRO_SPI_ICM20689) || defined(USE_GYRO_L3GD20)
 
     if (!mpuDetect(&gyroSensor->gyroDev, config)) {
+#if !defined(USE_FAKE_GYRO) // Allow resorting to fake accgyro
```

Comment:
Why so skimpy? You can as well put the conditional around the `if` as well. :-P

Reply:
We still want `mpuDetect` to run even if `USE_FAKE_GYRO` is defined. This way, we only have to insert a `USE_FAKE_GYRO` line and add a driver file to `.mk` for an actual target to run on bare dev boards and make it arm-able.

Reply:
Ah, that's better.